src/index.js

```diff
@@ -10,7 +10,19 @@ export function WebpackPnpExternals(options={}) {
             throw err;
         pnpApi = null;
     }
-    return (context, request, callback) => {
+    return function() {
+        const arg1 = arguments[0];
+        const arg2 = arguments[1];
+        const arg3 = arguments[2];
+        let context = arg1;
+        let request = arg2;
+        let callback = arg3;
+        // in case of webpack 5
+        if (arg1 && arg1.context && arg1.request) {
+            context = arg1.context;
+            request = arg1.request;
+            callback = arg2;
+        }
```

Comment:
I'd prefer something like the following:
```js
return function(context, request, callback){
    if(arguments.length === 2)
        [{context, request}, callback] = arguments;
```
It's more concise and IMO more readable.

Reply:
Sounds reasonable! I just copied the code from `webpack-node-externals`, but I can definitely make this change.